lib/authzed/api/v0/client.rb

```diff
@@ -38,14 +38,14 @@ def initialize(target:, credentials: nil, interceptors: [], options: {}, timeout
             channel_args: options,
           )
         end
-      end
 
-      class User
-        def initialize(namespace:, object_id:)
-          Authzed::Api::V0::ObjectAndRelation.new(
-            namespace: namespace,
-            object_id: object_id,
-            relation: '...',
+        def self.user(namespace:, object_id:)
```

Comment:
Moved to a helper method to avoid colliding with the generated User pb class

Reply:
I think we should just drop it all-together then. We can readdress when we start getting lots of feedback on the ruby client.

Reply:
What about patching the User class with the helper method which is a familiar ruby pattern:
```
Authzed::Api::V0::User.for(namespace: 'foo', object_id: 'bar')
```
otherwise it's:
```
          Authzed::Api::V0::User.new(
            userset: Authzed::Api::V0::ObjectAndRelation.new(
              namespace: namespace,
              object_id: object_id,
              relation: '...',
            )
          )
```